cypress/e2e/cloud/usage.test.ts

```diff
@@ -142,7 +142,10 @@ describe('Usage Page PAYG With Data', () => {
     cy.getByTestID('single-stat')
       .should('have.length', 4)
       .each((child, index) => {
-        expect(child.text().trim()).to.equal(stats[index])
+        expect(child.text().trim()).to.be.oneOf([
+          stats[index],
+          Number(stats[index]),
+        ])
```

Comment:
weird issue where firefox doesn't render a comma when the tests are run in circle (but it works fine in real life). So as long as either the string value or numerical value match, we're good to go